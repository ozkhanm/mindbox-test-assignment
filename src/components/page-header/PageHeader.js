const PageHeader = () => {
    return (
        <header className="header">
            <h1 className="header-title">todos</h1>
        </header>
    );
};

export default PageHeader;
