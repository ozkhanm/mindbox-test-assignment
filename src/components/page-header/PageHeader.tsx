import React from "react";

const PageHeader: React.FC = () => {
    return (
        <header className="header">
            <h1 className="header-title">todos</h1>
        </header>
    );
};

export default PageHeader;
