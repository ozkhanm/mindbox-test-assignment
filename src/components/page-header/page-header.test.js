import React from "react";
import renderer from "react-test-renderer";

import PageHeader from "./PageHeader";

it(`PageHeader renders correctly`, () => {
    const tree = renderer
        .create(<PageHeader />)
        .toJSON();
  
    expect(tree).toMatchSnapshot();
});
