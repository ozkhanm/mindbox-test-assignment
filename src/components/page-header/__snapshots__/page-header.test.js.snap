// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader renders correctly 1`] = `
<header
  className="header"
>
  <h1
    className="header-title"
  >
    todos
  </h1>
</header>
`;
